### DISCUSSION TITLE
Questions about install

### DISCUSSION THREAD

**Discussion by liujam2001** [COMBINE-lab/salmon#894] [General]
Hi I tried to install Salmon in my Mac (M1), when I tried the condo install, it didn't find it: 

PackagesNotFoundError: The following packages are not available from current channels:

  - salmon

When I download the 'salmon-master' from here, I had this message during 'make' and 'make install' process.

configure: error: in `/Applications/salmon-master/external/staden-io_lib':
configure: error: C compiler cannot create executables

Could you point out the right thing to do for next step? Thanks!

**Comment by rob-p** [comment]
Hi @liujam2001,

  To use salmon on M1 (or M2) macs, you likely have to run it under rosetta2.  Basically, the idea is you create a rosetta2 terminal (where x86 executables can be run), and make a bioconda environment there with the tools you wish to install.  You can read more [here](https://github.com/bioconda/bioconda-recipes/issues/41702#issuecomment-1614173361).

Best,
Rob

**Comment by liujam2001** [comment]
by the way, these are two errors I got during 'make'
make[2]: *** [libstadenio-prefix/src/libstadenio-stamp/libstadenio-configure] Error 77
make[1]: *** [CMakeFiles/libstadenio.dir/all] Error 2

**Comment by liujam2001** [comment]
I got it work now. Just every time I need to go to the rosseta environment. Thanks Rob
